test/crypten/test_context.py

```diff
@@ -17,7 +17,7 @@ def test_context(workers):
 
     @run_multiworkers([alice, bob], master_addr="127.0.0.1")
     @sy.func2plan()
-    def plan_func():
+    def plan_func(crypten=crypten):
```

Comment:
Q: Why is this needed here?

Reply:
Currently, there is a wrapped framework needed (and that will track the operations) -- in the ```syft/execution/tracing.py```